demos/python_demos/image_retrieval_demo/image_retrieval_demo/roi_detector_on_video.py

```diff
@@ -0,0 +1,57 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+
+import cv2
+from image_retrieval_demo.roi_cv_detector.detect_by_simple_dense_optical_flow import RoiDetector, \
+    get_rect_tl, get_rect_br
+
+
+class RoiDetectorOnVideo:
+    """ This class detects moving ROI on videos. """
+
+    def __init__(self, path):
+        if not os.path.exists(path):
+            raise Exception('File not found: {}'.format(path))
+
+        self.cap = cv2.VideoCapture(path)
```

Comment:
It does not handle web cam, while help for `-i` claims `Path to a video file or numeric camera ID.`

Reply:
replaced by '.. or a device node of a web-camera.'